src/scvi/external/methylvi/_module.py

```diff
@@ -163,7 +163,7 @@ def inference(self, mc, cov, batch_index, cat_covs=None, n_samples=1):
         else:
             categorical_input = ()
 
-        qz, z = self.z_encoder(methylation_input, batch_index, *categorical_input)
+        qz, z = self.z_encoder(methylation_input, None, batch_index, *categorical_input)
```

Comment:
Can you add the None as an argument in line 148. It makes it cleaner where it comes from.

Reply:
This is throughout this PR.